compiler/src/iree/compiler/Codegen/Common/EmulateNarrowType.cpp

```diff
@@ -615,7 +621,6 @@ LogicalResult emulateNarrowType(
       arith::ArithDialect, vector::VectorDialect, memref::MemRefDialect,
       affine::AffineDialect, IREE::HAL::HALDialect>(opLegalCallback);
 
-  RewritePatternSet patterns(ctx);
```

Comment:
Can we revert the change and move the `populateFlatten..` right after this line? It looks easier if we can group all populating pattern methods together.